astroplan/core.py

```diff
@@ -161,6 +163,46 @@ def altaz(self, time, target=None, obswl=None):
                 coordinate = target
             return coordinate.transform_to(altaz_frame)
 
+    def parallactic_angle(self, time, target):
+        '''
+        Calculate the parallactic angle.
```

Comment:
Looks good to me. And one of the Astropy project leads is [to blame](https://github.com/astropy/astropy/blame/6d0e5b050c5b71be9049ec662143f5b55b2a9a41/astropy/coordinates/baseframe.py#L1094) for that, so it's probably the recommended format.